container/apptabs.go

```diff
@@ -422,22 +425,30 @@ func (r *appTabsRenderer) moveSelection() {
 		return
 	}
 
-	if r.animation != nil {
-		r.animation.Stop()
-	}
-	r.animation = canvas.NewPositionAnimation(r.underline.Position(), underlinePos, canvas.DurationShort, func(p fyne.Position) {
-		r.underline.Move(p)
-		canvas.Refresh(r.underline)
-		if p == underlinePos {
-			r.animation = nil
+	if withAnimation {
```

Comment:
If this was !withAnimation then we could handle the non-animated case and return, returning the main code to the least indentation.
This is our style but also makes it easier to merge back to develop after release.

Reply:
That makes sense, updated the PR.